go/vt/sqlparser/ast_format.go

```diff
@@ -868,9 +869,6 @@ func (node *AliasedTableExpr) Format(buf *TrackedBuffer) {
 	buf.astPrintf(node, "%v%v", node.Expr, node.Partitions)
 	if !node.As.IsEmpty() {
 		buf.astPrintf(node, " as %v", node.As)
-		if len(node.Columns) != 0 {
```

Comment:
Why do you think we should remove this?

Reply:
Didn't intend to -- must have been a merge issue. Re-adding it. Thanks!